datajoint/plugin.py

```diff
@@ -15,10 +18,10 @@ def _update_error_stack(plugin_name):
         signature = plugin_meta.get_metadata("{}.sig".format(plugin_name))
         pubkey_path = str(Path(base_meta.egg_info, "{}.pub".format(base_name)))
         verify(pubkey_path=pubkey_path, data=data, signature=signature)
-        print("DataJoint verified plugin `{}` detected.".format(plugin_name))
+        logger.info("DataJoint verified plugin `{}` detected.".format(plugin_name))
         return True
     except (FileNotFoundError, InvalidSignature):
-        print("Unverified plugin `{}` detected.".format(plugin_name))
+        logger.warning("Unverified plugin `{}` detected.".format(plugin_name))
```

Comment:
Let's switch to f-strings since we no longer support Python 3.6.
```suggestion
        logger.warning(f"Unverified plugin `{plugin_name}` detected.")
```

Reply:
sounds good, this is due to this PR relying on PR #1031, I will change it over in that PR.